### DISCUSSION TITLE
Does CodeQL support ReactJS & NodeJS languages

### DISCUSSION THREAD

**Discussion by kunalsd** [community/community#63360] [Enterprise]
Hi 
Wish to know if CodeQL supports ReactJS & NodeJS languages to perform code analysis & vulnerability check using Github Enterprise Cloud.

I could not locate ReactJS & Nodejs is this list:
https://codeql.github.com/docs/codeql-overview/supported-languages-and-frameworks/

Thanks
Kunal

**Comment by mvkaran** [comment]
Hi @kunalsd 👋🏼 

Node, React and React Native are supported as mentioned under the `JavaScript and TypeScript built-in support` on the same page.

<img width="861" alt="Screenshot 2023-08-10 at 3 31 58 PM" src="https://github.com/community/community/assets/8726608/f57fde91-8a5c-4841-ab4e-ff8da768c49d">
